### ISSUE TITLE
Update language mappings, following update in lila

### ISSUE THREAD

**Issue by superuser-does** [lichess-org/mobile#1318]
Hello, this is an informative request to advise some language mappings (`language-LOCALE.xml` files used by Lila) have had their codes updated. See my PR https://github.com/lichess-org/lila/pull/16671 for full details.

Unfortunately, I'm not familiar with how Lichess Mobile pulls translations, beyond the fact the same updates should probably be made in [`/lib/src/utils/l10n.dart`](https://github.com/lichess-org/mobile/blob/main/lib/src/utils/l10n.dart).

If you do update this file, please note also two more commits you may want to sync with:

1. https://github.com/lichess-org/lila/pull/16670 - where 6 languages were delisted from public view.
2. Asturian was also enabled per https://github.com/lichess-org/lila/commit/fba10165cc8adb57b2c5fcdbd2a27ea890e3b03b.

**Comment by veloce** [comment]
Ok I will update the mobile side according to your PR. Once it is done I will request a review from you if you don't mind.

**Comment by superuser-does** [comment]
Thanks, happy to review.

There have been 3 more keys changed since the above post, c.f. https://github.com/lichess-org/lila/pull/1673 - you will also want to remove any "first letter" from the results table containing inaccuracies etc, as it's now held in the string itself.

**Comment by veloce** [comment]
Actually the only impact was on Esperanto, because we're using a subset of the Lila langs.

We'd need to update the mobile language list at some point but I leave that to another issue.
